playbooks/operationalizing/ldap_group_sync.adoc

```diff
@@ -0,0 +1,341 @@
+---
+---
+= LDAP Group Synchronization
+Red Hat Containers & PaaS CoP
+v1.0, 2017-06-10
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
```

Comment:
Please double check the `scripts_repo` as we no longer should be using the `rhc-ose` repo for any of this.